notebook/base/handlers.py

```diff
@@ -380,7 +380,7 @@ def check_origin(self, origin_to_satisfy_tornado=""):
 
         # Check CORS headers
         if self.allow_origin:
-            allow = self.allow_origin == origin
+            allow = self.allow_origin == "*" or self.allow_origin == origin
```

Comment:
Is this change more for synergy with the change below?  Because it seems like [this code above](https://github.com/jupyter/notebook/blob/master/notebook/base/handlers.py#L359-L360) is addressing the wildcard.